src/features/PopularMovies/index.js

```diff
@@ -12,6 +12,7 @@ import {
   selectGenres,
 } from "./moviesSlice";
 import { useSelector, useDispatch } from "react-redux";
+import starIcon from "../../assets/Vector.svg";
```

Comment:
Don't you think you can call this icon a little different than Vector.svg? Maybe more specifically. Of course it's a trifle, but for the sake of consistency I think that name star-ico.svg will be ok. ;)